### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Form elements must have labels (#react-select-5-input)

### ISSUE THREAD

**Issue by SahilTara** [Microsoft/WebTemplateStudio#122]
**Issue**: `Form elements must have labels` ([`label`](https://dequeuniversity.com/rules/axe/3.1/label?application=axeAPI))

**Target application**: [Project Acorn](http://localhost:3000/#/SelectWebApp)
**All sidebar inputs need this**
**Element path**: #react-select-5-input

**Snippet**:

    <input id="react-select-5-input" readonly="" tabindex="0" class="css-14uuagi" value="">

**How to fix**:

    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty

**Environment**:
Chrome version 72.0.3626.121

====

This accessibility issue was found using Accessibility Insights for Web 2.0.4 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
